ko/api/odbc.rst

```diff
@@ -95,11 +95,12 @@ CUBRID ODBC Driver가 확인되었다면 응용 프로그램에서 접속하려
 * **Password** : 데이터베이스 사용자 비밀번호를 입력한다.
 * **Server Address** : 데이터베이스의 호스트 주소를 입력한다. **localhost** 또는 다른 서버의 IP 주소를 입력한다.
 * **Server Port** : CUBRID 브로커의 포트 번호를 입력한다. 기본값은 33000이다. CUBRID 브로커 포트 번호를 확인하려면 **cubrid_broker.conf** 파일에서 BROKER_PORT 값을 확인하거나, 터미널에서 cubrid service status 명령을 입력한다. 터미널에서 입력했을 때 화면은 다음과 같다.
-* **Charset** : 데이터베이스의 문자셋(charSet) 입력한다.
-* **Autocommit** : Autocommit mode를 입력한다. 기본값은 OFF이다.
 
   .. image:: /images/image80.png
 
+* **Charset** : 데이터베이스의 문자셋(charSet) 입력한다.
+* **Autocommit** : Autocommit mode를 입력한다. 기본값은 OFF이다.
+* **Omit Schema** : 버전 11.2 이상의 Engine에 연결하여 ODBC에서 단일 스키마 특징을 제공하는 기능이다. 이 기능은 **ERWin**:sup:`TM` 과 CUBRID 11.2 이상의 엔진을 연결하는 경우 필요하다 (omit_schema=yes). **YES** 인 경우 login 사용자가 소유한 테이블/뷰/Primary Key/Foreign Key만 제공되며, 이때 소유자의 이름은 제거된다. 기본값은 **NO** 이다
```

Comment:
```suggestion
* **Omit Schema** : 스키마 개념을 지원하는 11.2 버전 이상의 데이터베이스 서버에서 스키마 정보 관련 함수 사용시 로그인한 사용자가 생성한 스키마만을  제공하는 기능이다. 이 기능은 **ERWin**:sup:`TM` 이 CUBRID 11.2 버전 이상을 사용할 경우 필요하다 (omit_schema=yes). **YES** 인 경우 login 사용자가 소유한 테이블/뷰/Primary Key/Foreign Key만 제공되며, 이때 소유자의 이름은 제거된다. 기본값은 **NO** 이다
```

Reply:
It was difficult to describe exactly what I did.
Thank you for your comment!!